### DISCUSSION TITLE
Is there an inpaint model for sdxl in controlnet？

### DISCUSSION THREAD

**Discussion by rockhward** [Mikubill/sd-webui-controlnet#2654] [Q&A]
Is there an inpaint model for sdxl in controlnet？
sd1.5 can use inpaint in controlnet, but I can't find the inpaint model that adapts to sdxl

**Comment by 2blackbar** [comment]
yes in forge ui and fooocus ui
no it does not work with auto11 , yet

**Comment by zero41120** [comment]
The code commit on a1111 indicates that SDXL Inpainting is now supported. You can find more details here: [a1111 Code Commit](https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/14390).
